webapp/packages/core-utils/src/sanitizeHtml.test.tsx

```diff
@@ -0,0 +1,42 @@
+/*
+ * CloudBeaver - Cloud Database Manager
+ * Copyright (C) 2020-2024 DBeaver Corp and others
+ *
+ * Licensed under the Apache License, Version 2.0.
+ * you may not use this file except in compliance with the License.
+ */
+import { sanitizeHtml } from './sanitizeHtml';
+
+describe('sanitize', () => {
```

Comment:
i just wanted to test how library works and decided to left this in case we want to replace the lib with our own or other solution